### PR TITLE
Better than G feature

### PR DESCRIPTION
**Pull request by TylorShin**

### CODE REVIEW

File: app/components/paperShow/components/betterSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+import { History } from "history";
+import { connect, Dispatch } from "react-redux";
+import { withStyles } from "../../../helpers/withStylesHelper";
+import { Fos } from "../../../model/fos";
+import Icon from "../../../icons";
+import InputWithSuggestionList from "../../common/InputWithSuggestionList";
+import papersQueryFormatter from "../../../helpers/papersQueryFormatter";
+import ActionTicketManager from "../../../helpers/actionTicketManager";
+import { ACTION_TYPES } from "../../../actions/actionTypes";
+import { withRouter, RouteComponentProps } from "react-router-dom";
```
Comment (NohYujeong) [inline app/components/paperShow/components/betterSearch.tsx]: Please move this code upwards.

File: app/components/paperShow/components/betterSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  FOSList: Fos[];
+  suggestionKeywords: CompletionKeyword[];
+  isAnimated: boolean;
+  dispatch: Dispatch<any>;
+}
+
+function handleSubmitQuery(query: string, dispatch: Dispatch<any>, history: History) {
+  if (query.length < 2) {
+    return dispatch({
+      type: ACTION_TYPES.GLOBAL_ALERT_NOTIFICATION,
```
Comment (NohYujeong) [inline app/components/paperShow/components/betterSearch.tsx]: Why did not you use the alertToast helper?

Comment (TylorShin) [reply app/components/paperShow/components/betterSearch.tsx]: I'll remove that function later.
`alertToast` will be deprecated

Comment (NohYujeong) [reply app/components/paperShow/components/betterSearch.tsx]: OK!

**Pull request merged**